src/macros.rs

```diff
@@ -190,6 +190,22 @@ macro_rules! redis_module {
             use $crate::configuration::get_bool_default_config_value;
             use $crate::configuration::get_enum_default_config_value;
 
+            if ctx.is_null() {
+                return raw::Status::Err as _;
```

Comment:
We can log even with NULL context. If we can not log its better to crash with a huge panic here. If this will happened and exit silently we will spend hours trying to guess why the module did not start.